modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -116,32 +124,35 @@ For example:
 <properties> +
   <http.port>8081</http.port> +
 </properties>
-| replicationFactor (optional) | specifies the number of instances created for your application. +
+| replicationFactor (optional) | Specifies the number of instances created for your application. +
 The default value is `1`
-| publicUrl (optional) | specifies the Url of the deployed application.
-| lastMileSecurity (optional) | enables Last-Mile security to forward HTTPS connections to be decrypted by this application. +
-This requires an SSL certificate to be included in the Mule application and also requires more CPU resources. The default value is `false`.
-| clusteringEnabled (optional) | enable Mule clustering across each replica of the application. You must have at least two replicas of your application. The default value is `false`.
-| memoryReserved (optional) | defines the amount of memory allocated for each replica of the application. The default value is `700MB`. +
+| publicUrl (optional) | Specifies the URL of the deployed application.
+| lastMileSecurity (optional) | Enables Last-Mile security to forward HTTPS connections to be decrypted by this application. +
+This requires an SSL certificate to be included in the Mule application and also requires more CPU resources. 
+The default value is `false`.
+| clusteringEnabled (optional) | Enables Mule clustering across each replica of the application. 
+You must have at least two replicas of your application. The default value is `false`.
+| memoryReserved (optional) | Defines the amount of memory allocated for each replica of the application. 
```

Comment:
"..of the application." -> "of your application." for consistency w/ rest of the page.

Reply:
Got it - thanks!